src/app/main/component/user/components/profile/profile-dashboard/one-habit/one-habit.component.ts

```diff
@@ -51,6 +52,7 @@ export class OneHabitComponent implements OnInit, OnDestroy {
   ngOnInit() {
     this.currentDate = this.datePipe.transform(new Date(), 'yyy-MM-dd');
     this.buildHabitDescription();
+    this.fillArray();
```

Comment:
Why we don't just write `this.photoArrays.push(this.firstFriend, this.secondFriend);` here? Or simply declare this array like `photoArrays = [firstFriend, secondFriend]?
And are you sure that the number of task that is mentioned in the name of PR is the problem that is solved here?

Reply:
it is only one part of task. i will change it